PowerShell/JumpCloud Module/Private/NestedFunctions/Get-JCResults.ps1

```diff
@@ -147,6 +223,7 @@ Function Get-JCResults
                 }
                 else {
                     try {
+                        $ProgressPreference = "SilentlyContinue"
```

Comment:
What is the scope on ProgressPreference? Is it Per Function/ Per Session? I'm just wondering if we are setting it, do we need to un-set it in the end block of this function?

Reply:
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_preference_variables?view=powershell-7.2#progresspreference

It sounds like it's for the function, however I'm not 100% sure. It's not a functionality change compared to the default value other than removing any visual progress indication. It was particularly noticeable with successive API calls, and would actually cause some strange artifacting in the output (random linebreaks/whitespaces/etc)